libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/feedback/core/CarFeedbackSearchOptions.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.androidauto.car.feedback.core
+
+import com.mapbox.api.geocoding.v5.models.GeocodingResponse
+import com.mapbox.navigation.core.geodeeplink.GeoDeeplink
+import com.mapbox.search.record.FavoriteRecord
+import com.mapbox.search.result.SearchSuggestion
+
+data class CarFeedbackSearchOptions(
+    val favoriteRecords: List<FavoriteRecord>? = null,
+    val searchSuggestions: List<SearchSuggestion>? = null,
+    val geoDeeplink: GeoDeeplink? = null,
+    val geocodingResponse: GeocodingResponse? = null,
+)
```

Comment:
Would prefer that this `feedback.core` package is unaware of the search feedback options.

The search feedback options should live near the search feedback feature

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/5786

^ I'm trying to make each screen a removable feature. So if most of the search structures are near the search module, that will be easier.

Reply:
This can be implemented by splitting `CarFeedbackSender` into two classes, one for navigation feedback and another for search one. I can do that in a separate PR.